src/init.lua

```diff
@@ -1301,8 +1304,8 @@ xplr.config.modes.builtin.debug_error = {
         help = "open logs in editor",
         messages = {
           {
-            BashExec = [===[
-              cat "${XPLR_PIPE_LOGS_OUT:?}" | ${EDITOR:-vi} -
+            BashExec0 = [===[
+              cat "${XPLR_PIPE_LOGS_OUT:?}" | tr '\0' '\n'  | ${EDITOR:-vi} -
```

Comment:
Maybe you can help me understand how the logs pipe is used, but it seems like this should be newline delimited and any time filenames are used, they get shell quoted with the equivalent of `%q`.

Reply:
I think it's better to allow letting `\n` stay unescaped in the logs here, since the logs are most likely to be used by only humans, for debugging and reporting errors. So line breaks will help get better understanding of what's going on, without using any external tool to parse the logs.